### DISCUSSION TITLE
about debugger reload

### DISCUSSION THREAD

**Discussion by MrYZhou** [fastapi/fastapi#6271] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Python: FastAPI",
      "type": "python",
      "request": "launch",
      "module": "uvicorn",
      "args": [
        "main:app --reload"
      ],
      "jinja": true,
      "justMyCode": true
    }
  ]
}
```

### Description

when i launch project whit debugger. and the launch.json has config paramter --reload. but can not auto reload new code behavior

### Operating System

Windows

### FastAPI Version

0.79

### Python Version

3.10.5

**Comment by Kludex** [comment, marked as answer]
Duplicated: https://github.com/tiangolo/fastapi/discussions/6272
